### DISCUSSION TITLE
Corrupted segmented images in "recognition" tab

### DISCUSSION THREAD

**Discussion by NoX209** [jomjol/AI-on-the-edge-device#1596] [Q&A] [answered]
Hi,

in the "recognition" tab i got the following:
![image](https://user-images.githubusercontent.com/86572462/207943652-068a548e-f7de-46fe-bd02-32624593c847.png)

The captured image and the rectangles for the digits in blue are correct.
But i suspect that the 4 small images under "Digital Counter" should be cropped out versions from above. But somehow this looks like garbage.

What could the problem be?
- psram corruption
- sd card corruption
- firmware issues
- did i forgot something to configure

tested with firmware versions
- Release: v13.0.7 (Commit: 304b9e0)
- Development-Branch: rolling (Commit: 997bf0)

tested with 2 devices of
- ESP32 CAM V2 from arduino-projekte.info

tested with 2 sd cards
- 8GB Intenso
- 2GB SanDisk

Where does data of the segmented images from?
What could i change?

thx

**Comment by jomjol** [comment]
How long did you wait after the start? Did you have a full run already?

**Reply by NoX209** [reply]
I waited some minutes, while i was checking how far i come in this "test". I can't tell you an exact number.
But what i a "full run"?
How can i tell, if this full run was done?

**Reply by caco3** [reply]
Yes, a full run or round is going through the steps you listed.
During the alignment step, the image still is shown rotated.
A round is completed when the status sais "Flow finished"

**Comment by NoX209** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/86572462/207952345-9cf5d44f-99a9-485c-887b-5ab2ba7e08ba.png)

While restarting the device and wait a bit longer, i somehow got segmented images and a good recognition. nice.

So my advice would be:
- wait a bit longer
- or after a firmware update, disconnect your device from power for some time

thx for your time.

ps: what a "full run" is, would be nice to know. :)


EDIT:
after checking the configuration, i found a difference in the detected digits.
I created ROIs for 0965. But the recognition found 6570. It somehow rotated the image.
I will probably configure all available digits and try again.

EDIT2:
It seems that on the "overview" tab in the right lower corner there is a "Status" field.
Here are some example outputs i gathered:
- Status: Aligning (21:23:42)
- Status: Flow finished (21:24:58)
I think "Flow finished" is equal to "full run".
